system/HTTP/ContentSecurityPolicy.php

```diff
@@ -274,20 +272,20 @@ public function reportOnly(bool $value = true)
 	//--------------------------------------------------------------------
 
 	/**
-	 * Sets the base_uri value. Can be either a URI class or a simple string.
+	 * Adds a new base_uri value. Can be either a URI class or a simple string.
 	 *
 	 * base_uri restricts the URLs that can appear in a page’s <base> element.
 	 *
 	 * @see http://www.w3.org/TR/CSP/#directive-base-uri
 	 *
-	 * @param string  $uri
-	 * @param boolean $reportOnly
+	 * @param string|array $uri
+	 * @param boolean|null $override
 	 *
 	 * @return $this
 	 */
-	public function setBaseURI($uri, bool $reportOnly)
+	public function addBaseURI($uri, ?bool $override = null)
```

Comment:
Not a fan of renaming this to `$override`. In IDE's that provide hints on what the variable is, it showing `override` as the hint doesn't give any clue what you're overriding.

Reply:
Made some changes to the naming and the explanation. Does this make more sense now?

Reply:
Yes, I think that makes more sense. Though $reportOnly might have been clearer but what you have is cool :)